server/src/user/user.service.ts

```diff
@@ -21,4 +22,22 @@ export class UserService {
   async getUsers(): Promise<User[]> {
     return await this.userRepository.createQueryBuilder().limit(50).getMany();
   }
+
+  async updateUser(userId: string, body: UserDto): Promise<void> {
+    const user = this.userRepository.findOne(userId);
+
+    if (user) {
+      // Right now only updates isAuthorized and isAdmin, add more in the future as needed.
+      try {
+        await this.userRepository
```

Comment:
Should use 
```
this.userRepository.save({...user,
isAuthorized: true/false
})
```

Reply:
The FE will send either isAuthorized or isAdmin and not both. So if isAdmin is sent, isAuthorized will be empty and this empty will get updated to the database

Reply:
Hey Karthikk, not sure if I got it the idea correctly, but feel free to let me know.
I am not doing any explicit checks right now, like "if both are present - do nothing", basically just allowing the user to send one of either or both (simply making them optional).